contents/docs/integrate/ingestion-warnings.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: Ingestion warnings
+sidebarTitle: Ingestion warnings
+sidebar: Docs
+showTitle: true
+---
+
+Sometimes we run into problems during ingestion from suboptimal usage of PostHog, for example to to maintain data integrity we don't merge with generic IDs like `null`.
+If this happens we'll do our best to still ingest the event and we'll additionally log an ingestion warning. These warnings can be seen in the Data Management section.
+The rest of this page contains more details on the specific warnings.
+
+## Refused to merge with an illegal distinct id
+
+See [identifying users - considerations](/docs/integrate/identifying-users#considerations) for background.
```

Comment:
This would be easier to review as a single PR :sweat_smile: 

This is fine for now, just note I would perhaps combine this page and the one from https://github.com/PostHog/posthog.com/pull/4330/files. This probably requires a rewrite though - curious on @pjhul thoughts.

Reply:
Definitely agree here. What I'd actually recommend is to add this info as a section in the Data management manual page (since that's where it lives) and then link to it from the user properties page and live ingestion guide.

Reply:
I did the earlier one in the GH UI, aka single file edits (why separate PRs)